packages/x-date-pickers/src/internals/components/PickersToolbar.tsx

```diff
@@ -129,7 +121,10 @@ export const PickersToolbar = React.forwardRef(function PickersToolbar<TDate, TV
           ownerState={ownerState}
           color="inherit"
           data-mui-test="toggle-mobile-keyboard-view"
-          aria-label={getMobileKeyboardInputViewButtonText(isMobileKeyboardViewOpen, viewType)}
+          aria-label={localeText.toolbarInterfaceModifierButtonAriaLabel(
```

Comment:
Maybe naming it `inputModeToggleButtonAriaLabel` or `inputViewToggleButtonAriaLabel` would make the field name more clear to the consumer?

At the same time, renaming `isMobileKeyboardViewOpen` to `isKeyboardInputOpen` everywhere would also make sense, but, can of course be left for a future refactor. 😄